### ISSUE TITLE
URLs have change

### ISSUE THREAD

**Issue by Alex-Cook4** [quarks-edge/quarks.documentation#60]
The documentation URLs seem to have changed. For example: 

http://quarks-edge.github.io/quarks.documentation/docs/quarks/getinvolved/ became http://quarks-edge.github.io/quarks.documentation/mydoc/getinvolved.html

Can this be undone? Do we want "mydoc" to be part of the path?

**Comment by wmarshall484** [comment]
Yes, the template I used kept the markdown documents in a different location than before. I can look into changing it back if it's causing issues. Are you having problems with links?

**Comment by Alex-Cook4** [comment]
It's only an issue because we were pointing to documentation locations from the Quarks website. Since we're pulling the documentation into the site project at apache (https://issues.apache.org/jira/browse/QUARKS-2) I'm not sure if we should worry about fixing things here. I'm fine with new URLs, as long as that is intended. 

I would propose that we have the directory be called something other than "mydoc" so that the URL looks more official.

**Comment by chanskw** [comment]
I agree with @Alex-Cook4.  I think having a directory path called "mydoc" is strange.

The other thing we have to worry about is Google indexing. Changing the links like this causes google index to go out of date.  So, if people search on google, they may potentially get to an old link, and get a 404 error.  Any time we change URL like this, we should look at having some sort of redirect so old links are not broken.

**Comment by wmarshall484** [comment]
@chanskw, Since we're moving everything to the Apache site, aren't the Google index links out of date anyway? What we could do is make it so that when somebody follows a link from google to the github quarks.documentation site, it automatically redirects them to the correct page in the Apache site.

**Comment by wmarshall484** [comment]
> I think having a directory path called "mydoc" is strange.

Agreed, I'll fix that.

**Comment by Alex-Cook4** [comment]
@wmarshall484 +1 to the idea of automatically redirecting the github documentation site to Apache once we have that up.
